src/components/FilterMenu/FilterMenu.js

```diff
@@ -329,120 +329,128 @@ const FilterMenu = ({
                   return (
-                    <DatePicker
-                      date={filtersStore.dates.value[0]}
-                      dateTo={filtersStore.dates.value[1]}
-                      key={filter.type}
-                      label={filter.label}
-                      onChange={handleChangeDates}
-                      type="date-range-time"
-                      withOptions
-                    />
+                    <div className='filters-col' key={filter.type}>
+                      <DatePicker
+                        date={filtersStore.dates.value[0]}
+                        dateTo={filtersStore.dates.value[1]}
+                        label={filter.label}
+                        onChange={handleChangeDates}
+                        type='date-range-time'
+                        withOptions
+                      />
+                    </div>
                   )
                 case ITERATIONS_FILTER:
                   return (
-                    <CheckBox
-                      key={filter.type}
-                      item={{ label: filter.label, id: 'iter' }}
-                      onChange={handleIter}
-                      selectedId={filtersStore.iter}
-                    />
+                    <div className='filters-col' key={filter.type}>
+                      <CheckBox
+                        item={{ label: filter.label, id: 'iter' }}
+                        onChange={handleIter}
+                        selectedId={filtersStore.iter}
+                      />
+                    </div>
                   )
                 case SHOW_UNTAGGED_FILTER:
                   return (
-                    <CheckBox
-                      key={filter.type}
-                      className="filters-checkbox"
-                      item={{ label: filter.label, id: SHOW_UNTAGGED_ITEMS }}
-                      onChange={handleShowUntagged}
-                      selectedId={filtersStore.showUntagged}
-                    />
+                    <div className='filters-col' key={filter.type}>
+                      <CheckBox
+                        className='filters-checkbox'
+                        item={{ label: filter.label, id: SHOW_UNTAGGED_ITEMS }}
+                        onChange={handleShowUntagged}
+                        selectedId={filtersStore.showUntagged}
+                      />
+                    </div>
                   )
                 case PROJECT_FILTER:
                   return (
-                    <Select
-                      density="dense"
-                      className={''}
-                      label={filter.label}
-                      key={filter.type}
-                      onClick={project => handleSelectOption(project, filter)}
-                      options={filtersStore.projectOptions}
-                      selectedId={filtersStore.project}
-                    />
+                    <div className='filters-col' key={filter.type}>
```

Comment:
if all columns need to have this class, better to specify it one time in the map function. Rename it to `filter-column`